packages/gatsby-theme-aio/src/components/Menu/index.js

```diff
@@ -42,13 +42,30 @@ const Item = ({ children, isDivider = false, isHighlighted, isSelected, href = '
         text-align: left;
       `}
       {...props}>
+      <div css={css`display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    overflow: hidden;
+    ${ isHeightUnset && !isHeightUnset ? `margin-top: var(--spectrum-global-dimension-size-50) !important` : ""}
+}`}>
+        <div css={css`
+          width: 20px !important;
+         ${ isHeightUnset ? "" : "height: 15px !important;" }
+         margin-left: 2px !important;
```

Comment:
If you can add a padding-right: 5px would make the icon looks better to have some of space between the words.

Reply:
@louisachu  Added padding-right: 5px now it's looking fine